.circleci/config.yml

```diff
@@ -15,9 +15,10 @@ test-install-dependencies: &test-install-dependencies
     sudo apt-get install -y mongodb-org-shell google-chrome-stable
 
 test-run: &test-run
-  name: Run Tests
-  command: |
-    for i in $(seq 1 5); do mongo rocketchat --eval 'db.dropDatabase()' && npm test && s=0 && break || s=$? && sleep 1; done; (exit $s)
```

Comment:
*No PR* 

We decided our automated testing should run differently compared to RC: We don't want to retry the whole testsuite if one test failed, but we'll be executing the tests file-by-file.